src/renderer/components/layout/main-layout.module.scss

```diff
@@ -11,7 +11,7 @@
   grid-template-rows: [contents] 1fr [footer] auto;
   grid-template-columns: [sidebar] var(--sidebar-width) [contents] 1fr;
   width: 100%;
-  z-index: 1;
+  z-index: var(--z-index-base);
```

Comment:
Looks again that meaning of this should not correlate with some base, but something (some name) that works with `position: relative` to put child above another with `position: static` flow. 
So, maybe better `z-index: var(--z-index-above)` or give a better name to `parent:relative + child:above` combination. Or I'm missing something?